plugins/woocommerce/includes/admin/helper/class-wc-helper-admin.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * WooCommerce Admin Helper - React admin interface
+ *
+ * @package WooCommerce\Admin\Helper
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * WC_Helper Class
+ *
+ * The main entry-point for all things related to the Helper.
```

Comment:
It would be useful to add a line here about what the helper is for.

```suggestion
 * The main entry-point for all things related to the Helper.
 * The Helper manages the connection between the store and 
 * an account on WooCommerce.com.
```

Reply:
Agreed!